frontend/app_flowy/packages/flowy_editor/example/lib/plugin/image_node_widget.dart

```diff
@@ -26,7 +29,37 @@ class _ImageNodeWidget extends StatelessWidget {
     required this.editorState,
   }) : super(key: key);
 
-  String get src => node.attributes['image_src'] as String;
+  @override
+  State<_ImageNodeWidget> createState() => __ImageNodeWidgetState();
+}
+
+class __ImageNodeWidgetState extends State<_ImageNodeWidget> with Selectable {
+  Node get node => widget.node;
+  EditorState get editorState => widget.editorState;
+  String get src => widget.node.attributes['image_src'] as String;
+
+  @override
+  List<Rect> getSelectionRectsInSelection(Offset start, Offset end) {
+    final renderBox = context.findRenderObject() as RenderBox;
+    final size = renderBox.size;
+    final boxOffset = renderBox.localToGlobal(Offset.zero);
+    return [boxOffset & size];
+  }
+
+  @override
+  Rect getCursorRect(Offset start) {
+    final renderBox = context.findRenderObject() as RenderBox;
+    final size = Size(5, renderBox.size.height);
```

Comment:
Maybe using a constant variable to store the magic number is better.

Reply:
Yes, but the image_node_widget is just an example, so I use a number directly.